reaktive/src/commonMain/kotlin/com/badoo/reaktive/disposable/CompositeDisposable.kt

```diff
@@ -1,56 +1,32 @@
 package com.badoo.reaktive.disposable
 
-import com.badoo.reaktive.utils.atomic.AtomicReference
-import com.badoo.reaktive.utils.atomic.getAndSet
-import com.badoo.reaktive.utils.atomic.getAndUpdate
-
 /**
  * Thread-safe collection of [Disposable]
  */
-class CompositeDisposable : Disposable {
-
-    private val list = AtomicReference<List<Disposable>?>(emptyList())
-    override val isDisposed: Boolean get() = list.value == null
+@Suppress("EmptyDefaultConstructor")
+expect class CompositeDisposable() : Disposable {
```

Comment:
Why these changes in this diff?

Reply:
Hello @CherryPerry , there was an error when I did a `git merge` trying to pull in new commits from this original repo into my fork. I had to fix the merge errors. Currently, there is no difference as I fixed all errors.